repaygithub/cactus
# CACTUS-405 :: IE relatedTarget Support

https://repayonline.atlassian.net/browse/CACTUS-405

Should be a quick & easy one. I tested other usages of `relatedTarget` that I found, but I didn't notice any problems. If you happen to notice anything I missed, let me know!

### Testing
Test the following in both IE and your preferred browser
1. Open the Select "with comboBox" story and focus the combobox input. Click outside the select and make sure the list closes.
2. Open the Layout story and click the gear icon in the action bar. Click outside the panel to make sure it closes when focus leaves the panel.
